Forward fix for D84506893

larryliu0820 opened the pull request:
Summary: As titled.

Reviewed By: JacobSzwejbka

Differential Revision: D84525208

JacobSzwejbka reviewed (approved):
Review automatically exported from Phabricator review in Meta.

larryliu0820 committed:
Forward fix for D84506893 (#15068)

Summary:

As titled.

Reviewed By: JacobSzwejbka

Differential Revision: D84525208
